eggo/dag.py

```diff
@@ -96,13 +132,13 @@ def _dnload_to_local_upload_to_s3(source, destination, compression):
 
         # 3. upload to tmp S3 location
         tmp_s3_path = os.path.join(EGGO_S3_TMP_URL, random_id())
-        upload_cmd = 'pushd {tmp_dir} && aws s3 cp ./* {s3_path} && popd'
+        upload_cmd = 'pushd {tmp_dir} && aws s3 cp . {s3_path} --recursive && popd'
```

Comment:
globs weren't working for me with `aws s3 cp` when testing locally with the latest version of awscli, but `--recursive` worked fine. Didn't know you were purposely restricting nested folders. What version are you using?